tests/smoke-tests/.cfignore

```diff
@@ -0,0 +1,6 @@
+test/
+coverage/
+node_modules/
+jest.config.mjs
+tsconfig.json
+.env
```

Comment:
What about the dist folder? Does that have any impact?

Reply:
I want it to be pushed explicitly, I only copy the dist folder at the moment